node_eval/__init__.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+
+"""Evaluate simple JavaScript expressions. Use it with care every keystroke triggers an evaluation."""
+
+import os
+import subprocess
+from albertv0 import *
+
+__iid__ = 'PythonInterface/v0.1'
+__prettyname__ = 'Node Eval'
+__version__ = '1.0'
+__trigger__ = 'node '
+__author__ = 'Hammed Oyedele'
+__dependencies__ = ['node']
```

Comment:
please check the existance of the binary like so https://github.com/albertlauncher/python/blob/dev/units/__init_.py#L25-L26